patches/chrome-android-java-src-org-chromium-chrome-browser-bookmarks-BookmarkBridge.java.patch

```diff
@@ -0,0 +1,27 @@
+diff --git a/chrome/android/java/src/org/chromium/chrome/browser/bookmarks/BookmarkBridge.java b/chrome/android/java/src/org/chromium/chrome/browser/bookmarks/BookmarkBridge.java
+index bab4cb91f75cc8a8f3b238d6e658df93b8980af8..e987a732388af5965f69bc0ea1ef7104607e539c 100644
+--- a/chrome/android/java/src/org/chromium/chrome/browser/bookmarks/BookmarkBridge.java
++++ b/chrome/android/java/src/org/chromium/chrome/browser/bookmarks/BookmarkBridge.java
+@@ -39,6 +39,7 @@ import org.chromium.components.power_bookmarks.PowerBookmarkMeta;
+ import org.chromium.components.power_bookmarks.PowerBookmarkType;
+ import org.chromium.components.power_bookmarks.ShoppingSpecifics;
+ import org.chromium.content_public.browser.WebContents;
++import org.chromium.ui.base.WindowAndroid;
+ import org.chromium.url.GURL;
+ 
+ import java.util.ArrayList;
+@@ -1004,6 +1005,7 @@ class BookmarkBridge {
+         }
+         return pairList;
+     }
++    @CalledByNative public void bookmarksImported(boolean isSuccess) {} @CalledByNative public void bookmarksExported(boolean isSuccess) {}
```

Comment:
Why don't we move it to `BraveBookmarkBridge` and call `Java_BraveBookmarkBridge_bookmarksImported` instead?

Reply:
I have tried that but it is giving this error: `error: use of undeclared identifier 'Java_BraveBookmarkBridge_bookmarksImported'`

Reply:
DM-ed @tapanmodh with option how to eliminate this patch